app/src/handlers/PlayerHandler.ts

```diff
@@ -0,0 +1,37 @@
+import {injectable} from 'inversify';
+import 'reflect-metadata';
```

Comment:
`reflect-metadata` is already imported in the `bootstrap.ts` file.

Reply:
I needed this to run everything. Maybe with the changes to bootstrap.ts, I can get it to work